src/languages/clojure.js

```diff
@@ -2,43 +2,45 @@
 Language: Clojure
 Description: Clojure syntax (based on lisp.js)
 Author: mfornos
-Contributors: Martin Clausen <martin.clausene@gmail.com>
+Contributors: Martin Clausen <martin.clausene@gmail.com>; Alexandre Grison <a.grison@gmail.com>
 Website: https://clojure.org
 Category: lisp
 */
 
 function(hljs) {
+  var globals = 'def defonce defprotocol defstruct defmulti defmethod defn defn- defmacro deftype defrecord';
```

Comment:
Missing a space here.

If these so different might one of the other keyword classes be more appropriate also?